api/src/test/java/org/apache/iceberg/expressions/TestNumericLiteralConversions.java

```diff
@@ -18,155 +18,139 @@
  */
 package org.apache.iceberg.expressions;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.data.Offset.offset;
+
 import java.math.BigDecimal;
 import java.util.stream.IntStream;
 import org.apache.iceberg.types.Types;
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 public class TestNumericLiteralConversions {
   @Test
   public void testIntegerToLongConversion() {
     Literal<Integer> lit = Literal.of(34);
     Literal<Long> longLit = lit.to(Types.LongType.get());
 
-    Assert.assertEquals("Value should match", 34L, (long) longLit.value());
+    assertThat((long) longLit.value()).isEqualTo(34L);
   }
 
   @Test
   public void testIntegerToFloatConversion() {
     Literal<Integer> lit = Literal.of(34);
     Literal<Float> floatLit = lit.to(Types.FloatType.get());
 
-    Assert.assertEquals("Value should match", 34.0F, floatLit.value(), 0.0000000001D);
+    assertThat(floatLit.value()).isCloseTo(34.0F, offset(0.0000000001F));
   }
 
   @Test
   public void testIntegerToDoubleConversion() {
     Literal<Integer> lit = Literal.of(34);
     Literal<Double> doubleLit = lit.to(Types.DoubleType.get());
 
-    Assert.assertEquals("Value should match", 34.0D, doubleLit.value(), 0.0000000001D);
+    assertThat(doubleLit.value()).isCloseTo(34.0D, offset(0.0000000001D));
   }
 
   @Test
   public void testIntegerToDecimalConversion() {
     Literal<Integer> lit = Literal.of(34);
 
-    Assert.assertEquals(
-        "Value should match", new BigDecimal("34"), lit.to(Types.DecimalType.of(9, 0)).value());
-    Assert.assertEquals(
-        "Value should match", new BigDecimal("34.00"), lit.to(Types.DecimalType.of(9, 2)).value());
-    Assert.assertEquals(
-        "Value should match",
-        new BigDecimal("34.0000"),
-        lit.to(Types.DecimalType.of(9, 4)).value());
+    assertThat(lit.to(Types.DecimalType.of(9, 0)).value()).isEqualTo(new BigDecimal("34"));
+
```

Comment:
nit: can we remove these unnecessary newlines?

Reply:
removed the new line